src/content/docs/apm/agents/net-agent/net-agent-api/net-agent-api.mdx

```diff
@@ -0,0 +1,2579 @@
+
+* Customizing your app name
+* Creating custom transaction parameters
+* Reporting custom errors and metrics
+
+You can also customize some of the .NET agent's default behavior by adjusting [configuration settings](/docs/agents/net-agent/configuration/net-agent-configuration) or using [custom instrumentation](/docs/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation).
+
+## Requirements
+
+To use the .NET agent API, make sure you have the [latest .NET agent release](/docs/release-notes/agent-release-notes/net-release-notes). Then, add a reference to the agent in your project using one of the two options below:
+
+   * Add a reference to `NewRelic.Api.Agent.dll` to your project.
+    OR
+   * View and download the API package from the [NuGet Package Library](https://www.nuget.org/packages/NewRelic.Agent.Api/).
+
+## List of API calls
+
+   The following list contains the different calls you can make with the API, including syntax, requirements, functionality, and examples:
+
+<CollapserGroup>
+    <Collapser
+        id="DisableBrowserMonitoring"
+        title="DisableBrowserMonitoring"
+    >
+    ### Syntax
+
+        ```cs
+        NewRelic.Api.Agent.NewRelic.DisableBrowserMonitoring([boolean $override])
+        ```
+
+        Disable automatic injection of browser monitoring snippet on specific pages.
+
+    ### Requirements
+
+    * Compatible with all agent versions.
+    * Must be called inside a [transaction](/docs/glossary/glossary/#transaction).
+
+    ### Description
+
+    Add this call to disable the <DoNotTranslate>**automatic**</DoNotTranslate> injection of [<InlinePopover type="browser" />](/docs/browser/new-relic-browser/getting-started/new-relic-browser) scripts on specific pages. You can also add an optional override to disable <DoNotTranslate>**both**</DoNotTranslate> manual and automatic injection. In either case, put this API call as close as possible to the top of the view in which you want browser disabled.
+
+    <Callout variant="tip">
+    Compare [`GetBrowserTimingHeader()`](/docs/agents/net-agent/net-agent-api/getbrowsertimingheader-net-agent), which <DoNotTranslate>**adds**</DoNotTranslate> the browser script to the page.
+    </Callout>
+
+    ### Parameters
+
+    <table>
+    <thead>
+        <tr>
+        <th width="25%">
+            Parameter
+        </th>
+
+        <th>
+            Description
+        </th>
+        </tr>
+    </thead>
+
+    <tbody>
+        <tr>
+        <td>
+            `$override`
+
+            _boolean_
+        </td>
+
+        <td>
+            Optional. When `true`, disables all injection of browser scripts. This flag affects both manual and automatic injection. This also overrides the [`GetBrowserTimingHeader()`](/docs/agents/net-agent/net-agent-api/getbrowsertimingheader-net-agent) call.
+        </td>
+        </tr>
+    </tbody>
+    </table>
+
+    ### Examples
+
+    #### Disable automatic injection [#automatic-only]
```

Comment:
These IDs dont work in the preview build, I think b/c h4s arent really supported/preferred 
<img width="437" alt="image" src="https://github.com/newrelic/docs-website/assets/55203603/204d7cc3-d16d-4079-adfa-0de5eef76fdd">